src/services/cache-config/CacheConfigService.ts

```diff
@@ -374,7 +386,6 @@ export class CacheConfigService {
     this.updateHelper(builder, address.concat("partition-handling", "PARTITION_HANDLING"), config["partition-handling"]);
     this.updateHelper(builder, address.concat("transaction", "TRANSACTION"), config.transaction);
     this.updateHelper(builder, address.concat("state-transfer", "STATE_TRANSFER"), config["state-transfer"]);
-    this.updateHelper(builder, address.concat("backup", "BACKUP"), config.backup);
```

Comment:
@vblagoje We don't allow backups to be updated in the console, so it doesn't make sense to add the extra dmr step if we know nothing will have changed. Or am I missing something?